adserver/management/commands/archive_offers.py

```diff
@@ -93,10 +94,11 @@ def handle_archive_day(self, day):
                 WHERE date >= '{day:%Y-%m-%d}' AND date < '{end_day:%Y-%m-%d}'
                 ORDER BY date
             ) TO STDOUT WITH CSV HEADER"""
-        with connection[settings.REPLICA_SLUG].cursor() as cursor:
+        with connections[settings.REPLICA_SLUG].cursor() as cursor:
```

Comment:
I'm not 100% sure how this was working before...

Reply:
I patched it locally heh